runtime/log.go

```diff
@@ -0,0 +1,26 @@
+package runtime
+
+import (
+	"fmt"
+
+	"github.com/Sirupsen/logrus"
+)
+
+// Create a logger that can be passed around through the environment.
```

Comment:
Start docs comments like this with `CreateLogger ....` I think go fmt or one of its friends keeps telling me so...